src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -1,91 +1,145 @@
-import PAYMENT_CHECK_REF_NUM_INFO from '@salesforce/schema/DataImport__c.Payment_Check_Reference_Number__c';
-import PAYMENT_METHOD_INFO from '@salesforce/schema/DataImport__c.Payment_Method__c';
-import DI_ACCOUNT1_IMPORTED_INFO from '@salesforce/schema/DataImport__c.Account1Imported__c';
-import DI_CONTACT1_IMPORTED_INFO from '@salesforce/schema/DataImport__c.Contact1Imported__c';
-import DI_DONATION_DONOR_INFO from '@salesforce/schema/DataImport__c.Donation_Donor__c';
+import DONATION_AMOUNT_INFO
+    from '@salesforce/schema/DataImport__c.Donation_Amount__c'
+import DONATION_DATE_INFO
+    from '@salesforce/schema/DataImport__c.Donation_Date__c'
+import DONATION_CAMPAIGN_SOURCE_FIELD
+    from '@salesforce/schema/DataImport__c.DonationCampaignImported__c'
+import PAYMENT_CHECK_REF_NUM_INFO
+    from '@salesforce/schema/DataImport__c.Payment_Check_Reference_Number__c'
+import PAYMENT_METHOD_INFO
+    from '@salesforce/schema/DataImport__c.Payment_Method__c'
+import DI_ACCOUNT1_IMPORTED_INFO
+    from '@salesforce/schema/DataImport__c.Account1Imported__c'
+import DI_CONTACT1_IMPORTED_INFO
+    from '@salesforce/schema/DataImport__c.Contact1Imported__c'
+import DI_DONATION_DONOR_INFO
+    from '@salesforce/schema/DataImport__c.Donation_Donor__c'
 // Additional schema needed for donation donor validation
-import DI_ACCOUNT1_NAME_INFO from '@salesforce/schema/DataImport__c.Account1_Name__c';
-import DI_CONTACT1_LAST_NAME_INFO from '@salesforce/schema/DataImport__c.Contact1_Lastname__c';
-
-import CONTACT_INFO from '@salesforce/schema/Contact';
-import ACCOUNT_INFO from '@salesforce/schema/Account';
-
-import CONTACT_FIRST_NAME_INFO from '@salesforce/schema/Contact.FirstName';
-import CONTACT_LAST_NAME_INFO from '@salesforce/schema/Contact.LastName';
-import ACCOUNT_NAME_INFO from '@salesforce/schema/Account.Name';
-
-import commonError from '@salesforce/label/c.commonError';
-import commonUnknownError from '@salesforce/label/c.commonUnknownError';
-
-import getDataImportSettings from '@salesforce/apex/UTIL_CustomSettingsFacade.getDataImportSettings';
-import getGiftEntrySettings from
-        '@salesforce/apex/GE_GiftEntry_UTIL.getGiftEntrySettings';
+import DI_ACCOUNT1_NAME_INFO
+    from '@salesforce/schema/DataImport__c.Account1_Name__c'
+import DI_CONTACT1_LAST_NAME_INFO
+    from '@salesforce/schema/DataImport__c.Contact1_Lastname__c'
+
+import CONTACT_INFO from '@salesforce/schema/Contact'
+import ACCOUNT_INFO from '@salesforce/schema/Account'
+
+import CONTACT_FIRST_NAME_INFO from '@salesforce/schema/Contact.FirstName'
+import CONTACT_LAST_NAME_INFO from '@salesforce/schema/Contact.LastName'
+import ACCOUNT_NAME_INFO from '@salesforce/schema/Account.Name'
+
+import getDataImportSettings
+    from '@salesforce/apex/UTIL_CustomSettingsFacade.getDataImportSettings'
+import getGiftEntrySettings
+    from '@salesforce/apex/GE_GiftEntry_UTIL.getGiftEntrySettings'
+import { fireEvent } from 'c/pubsubNoPageRef'
+import getNamespaceWrapper
```

Comment:
prefer getNamespace from utilCommon

Reply:
I think that makes sense, I'll move it to utilCommon

Reply:
Updated!